extensions/interactions/AlgebraicExpressionInput/protractor.js

```diff
@@ -30,13 +30,13 @@ var expectInteractionDetailsToMatch = async function(elem) {
   ).toBe(true);
   // Testing editor's value in default state.
   expect(
-    await objects.AlgebraicExpressionEditor(elem.element(by.tagName(
+    await objects.MathEditor(elem.element(by.tagName(
```

Comment:
I don't understand what's going on with the name changes throughout this PR. Could you add a description to the top comment explaining what these various editors and interactions do and what names are changing?

Reply:
Comment added. Also, specifically about this change -- The thing is, both the MathEquationInput and AlgebraicExpressionInput interactions use the same editor which is why I've used a single function in objects' protractor that can be used by both editors.

Reply:
Thanks for the clarification! This makes sense now